support/remote_health_monitoring/about-remote-health-monitoring.adoc

```diff
@@ -5,16 +5,66 @@ include::modules/common-attributes.adoc[]
 
 ifdef::openshift-origin[]
 {product-title} may be installed without a pull secret received at cloud.redhat.com. In this case default imagestreams will not be imported and telemetry data will not be sent.
 endif::[]
 
 include::modules/telemetry-about-telemetry.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See the xref:../../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[{product-title} update documentation] for more information about updating or upgrading a cluster.
+
 include::modules/telemetry-what-information-is-collected.adoc[leveloffset=+2]
 
+.Additional resources
+
+* See xref:../../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.adoc#showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by Telemetry] for details about how to list the attributes that Telemetry gathers from Prometheus in {product-title}.
+
+* See the link:https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_04-config.yaml[upstream cluster-monitoring-operator source code] for a list of the attributes that Telemetry gathers from Prometheus.
+
+* Telemetry is installed and enabled by default. If you need to opt out of remote health reporting, see xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting[Opting out of remote health reporting].
+
 include::modules/insights-operator-about.adoc[leveloffset=+1]
+
+.Additional resources
+
+* The Insights Operator is installed and enabled by default. If you need to opt out of remote health reporting, see xref:../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting[Opting out of remote health reporting].
+
 include::modules/insights-operator-what-information-is-collected.adoc[leveloffset=+2]
+
+.Additional resources
+
+* See xref:../../support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.html#insights-operator-showing-data-collected-from-the-cluster_showing-data-collected-by-remote-health-monitoring[Showing data collected by the Insights Operator] for details about how to review the data that is collected by Insights Operator.
+
+* The Insights Operator source code is available for review and contribution. See the link:https://github.com/openshift/insights-operator/blob/master/docs/gathered-data.md[Insights Operator upstream project] for a list of the items collected by Insights Operator.
+
+include::modules/understanding-telemetry-and-insights-operator-data-flow.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack_understanding-the-monitoring-stack[Understanding the monitoring stack] for more information about the {product-title} monitoring stack.
```

Comment:
* See https://docs.openshift.com/container-platform/4.5/installing/install_config/configuring-firewall.html#configuring-firewall for information about Firewall settings and enabling endpoints for Telemetry and Insights

Reply:
@radekvokal thank you for your review! This is a great suggestion. I have now added the cross-reference.